Se 5237 until successful

Cristian-Venticinque opened the pull request.

Commit:
Removed inline "See also" section with redundant links.

Commit:
Added headers to table.

Commit:
Added [NOTE] clarifying `failureExpression` checks.

Commit:
Properly formatted [NOTE] block.

Commit:
Properly formatted section title as H3.

Commit:
Removed unused adoc styling.

Commit:
Removed incorrect *bold* usage. Added `backticks` to values.

Commit:
Added `backticks` for SUCCESS and FAILURE values.

Commit:
Added `backticks` to class name.

valkyrie69 commented on modules/ROOT/pages/until-successful-scope.adoc:
```diff
@@ -134,60 +132,60 @@ See xref:mule-object-stores.adoc[Mule Object Stores] for further information abo
 
-*Note*: For an example of using *Until-Successful* in Anypoint Studio see the <<xmlexample,XML Example>> and the canvas flow thereafter.
+[NOTE]
+====
```
`until-successful`

valkyrie69 commented on modules/ROOT/pages/until-successful-scope.adoc:
```diff
@@ -210,50 +208,48 @@ To the until-successful element, add child element `threading-profile`. Configur
 |===
 |Attribute |Description
-|*maxBufferSize* |Determines how many requests are queued when the pool is at maximum usage capacity and the pool exhausted action is WAIT. The buffer is used as an overflow.
+|maxBufferSize |Determines how many requests are queued when the pool is at maximum usage capacity and the pool exhausted action is `WAIT`. The buffer is used as an overflow.
```
`maxBufferSize`

valkyrie69 commented on modules/ROOT/pages/until-successful-scope.adoc:
```diff
@@ -210,50 +208,48 @@ To the until-successful element, add child element `threading-profile`. Configur
-|*maxThreadsActive* |The maximum number of threads to use.
+Type: Integer +
+Required: No
+|maxThreadsActive |The maximum number of threads to use.
```
`maxThreadsActive` I won't comment on all of these, but these types should all have tick marks. 
I think the instances of "pool exhausted action" that are in plain text need to be changed to `poolExhaustedAction`.   You can S&R

valkyrie69 commented on modules/ROOT/pages/until-successful-scope.adoc:
```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 :keywords: anypoint studio, studio, mule, until successful, reattempts, retry
 
-The *until-successful* scope processes messages through its processors until the process succeeds. By default, until-successful's processing occurs asynchronously from the main flow. After passing a message into the until-successful scope, the main flow immediately regains control of the thread. However, you can configure until-successful to run synchronously relative to the main flow.
+The until-successful scope processes messages through its processors until the process succeeds. By default, until-successful's processing occurs asynchronously from the main flow. After passing a message into the until-successful scope, the main flow immediately regains control of the thread. However, you can configure until-successful to run synchronously relative to the main flow.
```
I think all instances of `until-successful` need the tick marks. I'm basing that on what I see Mysti doing for localization. There are way too many process words in the first sentence. I don't have suggestions though--I am hoping you are more creative than I am.

valkyrie69 commented on modules/ROOT/pages/until-successful-scope.adoc:
```diff
@@ -210,50 +208,48 @@ To the until-successful element, add child element `threading-profile`. Configur
-* DISCARD_OLDEST - Throw away the oldest request and return.
-* ABORT - Throw a RuntimeException.
-* RUN - (Default). The thread making the execute request runs the task itself, which helps guard against lockup.
+* `WAIT` - Wait until a thread becomes available; don't use this value if the minimum number of threads is zero, in which case a thread may never become available.
```
Use a period instead of the semi-colon. New sentence "Don't use this value..."

valkyrie69 commented on modules/ROOT/pages/until-successful-scope.adoc:
```diff
@@ -262,9 +258,6 @@ However, in some situations, you may need until-successful to process messages s
 
 When set to process message synchronously, until-successful executes within the thread of the main flow, then returns the result scope's processing on the same thread.
```
Until-Successful and until-successful (inconsistent caps)

valkyrie69 commented on modules/ROOT/pages/until-successful-scope.adoc:
```diff
@@ -262,9 +258,6 @@ However, in some situations, you may need until-successful to process messages s
-[discrete.view]
 === Studio Visual Editor
 
 In the *Threading* tab of the Until Successful's *Properties Editor*, click to select *Synchronous*.
```
This Until Successful...so many ways.

Commit:
Unified style used when referring to the Until Successful scope.

Whenever the documents mentions Until Successful, it is capitalized. 
On the other hand, when it is detailing the usage of the 
`until-successful` element in the XML, it was put into `backticks`.

Commit:
Added *bold* style to UI elements mentioned.

Commit:
Fixed punctuation.

Commit:
Added `backticks` to elements from the XML.
